gptables/core/api.py

```diff
@@ -75,7 +76,7 @@ def produce_workbook(
         if cover_gridlines:
             ws = wb.add_worksheet(cover.cover_label, gridlines=gridlines)
         else:
-            ws = wb.add_worksheet(cover.cover_label, gridlines=2)
+            ws = wb.add_worksheet(cover.cover_label, gridlines="hide_all")
```

Comment:
I don't think the if-else is necessary if we specify a default value in the function definition? Does this work if you just have `ws = wb.add_worksheet(cover.cover_label, gridlines=gridlines)` ?

Reply:
The if-else is for when you want gridlines on the workbook but not on the cover sheet specifically.